README.md

```diff
@@ -213,6 +213,36 @@ $ ./gradlew okcurl:nativeImage
 $ ./okcurl/build/graal/okcurl https://httpbin.org/get
 ```
 
+Java Modules
+------------
+
+OkHttp (5.3+) implements Java 9 Modules such that in Java builds.
+
+With this in place Java builds should fail if apps attempt to use internal packages.
+
+```
+error: package okhttp3.internal.platform is not visible
+    okhttp3.internal.platform.Platform.get();
+                    ^
+  (package okhttp3.internal.platform is declared in module okhttp3,
+    which does not export it to module i.am.bad.and.i.should.feel.bad)
```

Comment:
@swankjesse I guess this is the whole point?

Reply:
Also added a CI test that jlink works, see https://www.baeldung.com/jlink

Reply:
Also https://snyk.io/blog/jlink-create-docker-images-spring-boot-java/

Reply:
@swankjesse do you need more convincing?

Reply:
This is good behavior. I wouldn’t use `i.am.bad.and.i.should.feel.bad` as the example; this is not a place for shame